source/_cookbook/notify_if__new_ha_release.markdown

```diff
@@ -30,3 +30,24 @@ automation:
       data:
         message: 'There is a new Home Assistant release available.'
 ```
+
+You can use [templates](/topics/templating/) to include the release number of Home Assistant if you prefer. The following example sends a notification via [Pushbullet](/components/notify.pushbullet/) with the Home Assistant version in the message.
+
+```yaml
+notify:
+  platform: pushbullet
+  api_key: 'YOUR_KEY_HERE'
+  name: pushbullet
+
+automation:
+  - alias: Update notifications
+  trigger:
+    - platform: state
+      entity_id: updater.updater
+  action:
+    service: notify.pushbullet
+    data: 
+      title: 'New Home Assistant Release'
+      target: 'YOUR_TARGET_HERE' #See Pushbullet component for usage
+      message: "Home Assistant {{ states.updater.updater.state }} is now available."
+```
```

Comment:
Needs an empty line after for formatting to behave properly. With that exception this looks good!